### PR TITLE
Auto-update for packages related to 'healthcare-components-1.0.0-master-20201015-1'

### PR DESCRIPTION
**Pull request by MicrosoftHealthService**
Updates package 'Microsoft.Health.Abstractions' to version '1.0.0-master-20201015-1'
Updates package 'Microsoft.Health.Api' to version '1.0.0-master-20201015-1'
Updates package 'Microsoft.Health.Extensions.DependencyInjection' to version '1.0.0-master-20201015-1'
Updates package 'Microsoft.Health.Core' to version '1.0.0-master-20201015-1'
Updates package 'Microsoft.Health.Test.Utilities' to version '1.0.0-master-20201015-1'
Updates package 'Microsoft.Health.Extensions.BuildTimeCodeGenerator' to version '1.0.0-master-20201015-1'
Updates package 'Microsoft.Health.Client' to version '1.0.0-master-20201015-1'
Updates package 'Microsoft.Health.SqlServer' to version '1.0.0-master-20201015-1'
Updates package 'Microsoft.Health.SqlServer.Api' to version '1.0.0-master-20201015-1'

### CODE REVIEW

**Comment by rbans96** [comment]
@v-shaaal Should we update LogExecuted(HttpContext httpContext, IClaimsExtractor claimsExtractor) to also pass bool as the IAuditHelper is updated in the [PR](https://github.com/microsoft/healthcare-shared-components/pull/111/files)?

**Comment by v-shaaal** [comment]
> @v-shaaal Should we update LogExecuted(HttpContext httpContext, IClaimsExtractor claimsExtractor) to also pass bool as the IAuditHelper is updated in the [PR](https://github.com/microsoft/healthcare-shared-components/pull/111/files)?

Hi Richa,
yes, changes in FHIR will also be required to reflect the boolean. I have taken that as a work item and will be sending out a pull request hopefully today.

File: src/Microsoft.Health.Fhir.Api/Features/Audit/AuditHelper.cs [hunk 1]
```diff
@@ -44,13 +44,22 @@ public void LogExecuting(HttpContext httpContext, IClaimsExtractor claimsExtract
...
+        /// <param name="claimsExtractor">The extractor used to extract claims.</param>
+        /// <param name="shouldCheckForAuthXFailure">Only emit LogExecuted messages if this is an authentication error (401), since others would already have been logged.</param>
+        public void LogExecuted(HttpContext httpContext, IClaimsExtractor claimsExtractor, bool shouldCheckForAuthXFailure = false)
         {
             EnsureArg.IsNotNull(claimsExtractor, nameof(claimsExtractor));
             EnsureArg.IsNotNull(httpContext, nameof(httpContext));
 
-            Log(AuditAction.Executed, (HttpStatusCode)httpContext.Response.StatusCode, httpContext, claimsExtractor);
+            var responseStatusCode = (HttpStatusCode)httpContext.Response.StatusCode;
+            if (!shouldCheckForAuthXFailure || responseStatusCode == HttpStatusCode.Unauthorized)
```
Comment (v-shaaal) [inline src/Microsoft.Health.Fhir.Api/Features/Audit/AuditHelper.cs:59]: Thanks for making this change!

**Review by johnstairs — APPROVED** [review]

**Review by v-shaaal — APPROVED** [review]

**Pull request merged**